### ISSUE TITLE
Should we show "(copyleft) Unshared" in copyright date field for unshared projects?

### ISSUE THREAD

**Issue by benjiwheeler** [LLK/scratch-www#2446]
Seems to me that this looks pretty weird:

![screen shot 2018-12-11 at 4 18 19 pm](https://user-images.githubusercontent.com/3431616/49831035-db231f80-fd60-11e8-8da2-6e4fe52c02b7.png)

**Comment by Kenny2github** [comment]
It's not about the copyleft mark itself, it's just that it seems weird to show "unshared" where a date would appear. I'd say just hide it.

**Comment by thisandagain** [comment]
Agreed. I think we should just hide this when the project is unshared.
